calculator/src/test/java/com/market/banica/calculator/controllerTests/CalculatorControllerTest.java

```diff
@@ -44,13 +46,15 @@ void getRecipe() throws Exception {
         String product = "baklava";
         int quantity = 100;
 
-        RecipeDTO dummyRecipe = new RecipeDTO();
-        dummyRecipe.setIngredients(new HashSet<>());
+        List<ProductDto> productDtoList = new ArrayList<>();
+        ProductDto dummyRecipe = new ProductDto();
+        dummyRecipe.setIngredients(new HashMap<>());
         dummyRecipe.setItemName("baklava");
         dummyRecipe.setTotalPrice(BigDecimal.valueOf(10));
+        productDtoList.add(dummyRecipe);
 
 
-        given(service.getRecipe(clientId, product, 100)).willReturn(dummyRecipe);
+        given(service.getRecipe(clientId, product, 100)).willReturn(productDtoList);
```

Comment:
"100" could be extracted in constant.

Reply:
It is not part of my code, but from the merge commit with main

Reply:
It doesn't matter if you didn't write it initially, our goal is to improve the code quality whenever possible, so please change it

Reply:
Fixed